src/lib.rs

```diff
@@ -1,9 +1,4 @@
-use std::{
-    collections::HashMap,
-    env::current_dir,
-    fs,
-    path::{Path, PathBuf},
```

Comment:
looks like you dropped `Path` from the dependency here which is causing the wasm build to fail (see the CI checks), could you add this back? :)